core/src/main/scala/cats/Reducible.scala

```diff
@@ -112,6 +112,12 @@ import simulacrum.typeclass
       val F = self
       val G = Reducible[G]
     }
+
+  def minimum[A](fa: F[A])(implicit A: Order[A]): A =
```

Comment:
In this case you would have `Reducible[NonEmptyList].min(myThing)` how could there be a collision with the stdlib in practice? The collision could come with an implicit class that adds `.min` to any `Reducible`, and then we have anxiety, perhaps, when reading the code if we are getting the safe `Reducible` one or the stdlib (which would be safe in exactly the cases that the Reducible exists).

Is that what you mean?